package.json

```diff
@@ -9,7 +9,7 @@
   ],
   "description": "Use webpack with karma and fast, file source maps",
   "peerDependencies": {
-    "webpack": "^1.4.0 || ^2.1.0-beta"
+    "webpack": "^1.4.0 || ^2 || ^2.1.0-beta || ^2.2.0-rc"
```

Comment:
is `^2` the same as `^2.0.0`? I'd probably rather have that be explicitly `^2.0.0` so there's no confusion.

Also, have you tested this on 2.2 rc? If so, great, I had had a report that it did not work, but maybe it was just because of this.